easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-3.4.3-intel-2016b-Python-2.7.12-vsc-base-2.5.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'vsc-mympirun'
+version = '3.4.3'
+
+homepage = 'https://github.com/hpcugent/vsc-mympirun'
+description = """VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['https://github.com/hpcugent/vsc-mympirun/archive/']
+
+vsc_base_ver = '2.5.1'
+versionsuffix = '-Python-%%(pyver)s-vsc-base-%s' % vsc_base_ver
```

Comment:
the suffix was added because there was an existing easyconfig with another `vsc-base` version, and the need arose for a `vsc-mympirun` (same version) with a newer `vsc-base`